--- a/CMake/3rdparty.cmake	2025-08-01 13:14:04.298815621 -0400
+++ b/CMake/3rdparty.cmake	2025-08-01 13:14:16.926824534 -0400
@@ -38,11 +38,11 @@
       if(NOT JPEG_FOUND)
         message(STATUS "Info: DCMTK TIFF support will be enabled (but without JPEG)")
         include_directories(${TIFF_INCLUDE_DIR})
-        set(LIBTIFF_LIBS ${TIFF_LIBRARIES} ${TIFF_EXTRA_LIBS_STATIC})
+        set(LIBTIFF_LIBS TIFF::tiff)
       else()
         message(STATUS "Info: DCMTK TIFF support will be enabled")
         include_directories(${TIFF_INCLUDE_DIR} ${JPEG_INCLUDE_DIR})
-        set(LIBTIFF_LIBS ${TIFF_LIBRARIES} ${TIFF_EXTRA_LIBS_STATIC} ${JPEG_LIBRARIES})
+        set(LIBTIFF_LIBS TIFF::tiff ${JPEG_LIBRARY})
       endif()
     endif()
   endif()

